silnlp/nmt/translate.py

```diff
@@ -35,6 +62,52 @@ def _insert_lang_tag(self, text: str, trg_iso: Optional[str]) -> str:
             return f"<2{trg_iso}> {text}"
         return text
 
+    def translate_book_by_step(self, book, src_project=None, output_usfm=None, trg_lang=None):
```

Comment:
I would name this `translate_book`, which you should be able to do once it is a separate class from `NMTTranslator`. Also, please add typing info for all parameters.

Reply:
Done.